### ISSUE TITLE
NVDA not reading Live Region in Teams Admin portal (not announcing error message displayed on activating 'Save' button)

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13034]
### Steps to reproduce:
1. Open https://admin.teams.microsoft.com/devices/roomsystems
2. Navigate left rail->select Devices->Teams Rooms->Navigate to table->select offline->view details->Peripherals->Room camera->More option->Manage health impact.
3. Now activate Save button and observe the screen reader announcement.

If you do not have access to the above, here is a code snippet you can use:

```
<div
          role='alert'
          key={index} className={`ms-fadeIn100${hideIcon}`}>
          <MessageBarComponent
            messageBarType={item.messageType}
          >
... 
... 
</div> 
```

### Actual behavior:
NVDA is not announcing the status   message displayed on screen on activating save button.

### Expected behavior:
Screen reader should announce the status message "Health impact was updated for 1 peripheral" which is displayed on the screen on activating save button.

#### NVDA version:
2021.2

#### Windows version:
Windows 11, 21H2, Build 22000.258

#### Name and version of other software in use when reproducing the issue:
Chrome version: Version 94.0.4606.61 (Official Build) (64-bit)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Unknown

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Unknown

**Comment by josephsl** [comment]
Hi,

It's been six months since we last heard from the original poster. Has there been any updates since then?

Thanks.

**Comment by msftedad** [comment]
No changes from our side. Should we expect NVDA to have resolved this bug?

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

Kind Regards,
NV Access Software Developers

**Comment by rrjamesjr** [comment]
I have a similar issue except its related to role="status" in my live region. It works when I set to role="alert" but does not read anything when role="status".

`<span role="status">Status update</>` does not get announced when a toggle is switched but `<span role="alert">Status update</>`  does

**Comment by Adriani90** [comment]
I think here the role?"alert" must be set on the message bar and not on the div. Then the screen reader would read it out.
@msftedad are you still having this issue?

**Comment by msftedad** [comment]
Hi @Adriani90 Issue seems fixed in latest build of NVDA.

**Comment by Adriani90** [comment]
Thanks for the update. I am closing as works for me.
